app/controllers/chapel-credits.js

```diff
@@ -0,0 +1,149 @@
+                    if ($(window).innerWidth() < 768) {
+                        $(elements[i]).slideUp();
+                        console.log("twice");
+                    }
+                }
+                let form = $(item.target);
+                this.set("lastForm", form);
+            } else {
+                if (lastForm) {
+                    $(lastForm).removeClass("onclickOrange");
+
+                    let elements = $(lastForm).nextAll();
+                    for (var i = 0; i < 3; i++) {
+
+                        if ($(window).innerWidth() < 768) {
+                            $(elements[i]).slideUp();
+                            console.log("once");
+                        }
+                    }
+                }
+                let form = $(item.target);
+                $(form).addClass("onclickOrange");
+                let elements = $(form).nextAll();
+                for (var i = 0; i < 3; i++) {
+                    if ($(window).innerWidth() < 768) {
+                        $(elements[i]).slideDown();
+                        console.log("twice");
+                    }
+                }
+                this.set("lastForm", form);
+            }
+        },
+
+        toggleEventDetailsModal(item) {
+
+            $("#toggleEventDetailsModal").addClass("event-showModal");
+            $('.container').addClass('blur');
+            let context = this;
+            let displayEvent = this.set("displayEvent", item);
+            let chEventID = this.get("displayEvent.CHEventID", item);
+
+        },
+
+        filterEvents: function() {
+            // Filter the list of activities shown when user types in the search bar
+            let searchValue = this.get("model.searchValue");
+            if (searchValue) {
+                let newList = [];
+                let oldList = this.get("model.chapelEvents");
+                for (let i = 0; i < oldList.length; i++) {
+                    if (oldList[i].CHEventID.toLowerCase().indexOf(searchValue.toLowerCase()) !== -1) {
+                        newList.push(oldList[i]);
+                    } else if (oldList[i].CHDate.toLowerCase().indexOf(searchValue.toLowerCase()) !== -1) {
+                        newList.push(oldList[i]);
+                    } else if (oldList[i].CHDate.toLowerCase().indexOf(searchValue.toLowerCase()) !== -1) {
+                        newList.push(oldList[i]);
+                    } else if (oldList[i].CHTime.toLowerCase().indexOf(searchValue.toLowerCase()) !== -1) {
+                        newList.push(oldList[i]);
+                    }
+
+                }
+                this.set("model.eventShown", newList);
+            } else {
+                this.set("model.eventShown", this.get("model.chapelEvents"));
+            }
+        },
+
+        cancelEventDetailsModal(item) {
+
+            if (!($(item.target).hasClass("modal-content") || $(item.target).hasClass("modal-body") || $(item.target).hasClass("modal-footer"))) {
```

Comment:
I found a better way to do this. 
 `if(item.target == $(".profile-picture-cancel-button")[0] || item.target == $("#editProfilePictureModal").not(".modal-content")[0]){`
.profile-picture-cancel-button is a cancel button.
#editProfilePictureModal is the entire modal
.modal-content is just the visible modal content.  
This works much better if you have a lot of elements in the modal.